GameProject/Assets/Turn based assets/Scripts/CubeGrower.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using Turn_based_assets.Scripts;
+using UnityEngine;
+
+public class CubeGrower : MonoBehaviour, ISelection
+{
+    public void Select()
+    {
+        this.transform.localScale = this.transform.localScale * 3;
```

Comment:
It'd probably be good to have a `const` for the size factor. Have it one place, change it on place.

```cs
private const int ScaleFactor = 3;

public void Select()
{
    this.transform.localScale = this.transform.localScale * ScaleFactor;
}
```

Not sure if you guys have a naming convention for constants, just went for pascal here.

Reply:
I think this might have been done in another branch already, there's been a lot of duped work amongst a couple of branches